The warning messages are not informative after creating a folder or a file with the same name
kind/bug 
Reproduction Steps for a folder: Create any project. In the Project create a folder. Try to create a folder with the same name again. Expected behavior: The "Error" widget appears with the message "Resource already exists"(see screenshot 1). Observed behavior: The "Error" widget appears without the message about the folder exists. Reproduction Steps for a file: Create any project. In the Project create a file. Try to create a file with the same name again. Expected behavior: The "Error" notification appears with the message "Resource already exists"(see screenshot 3). Observed behavior: The "Error" notification appears without the message about the file exists(see screenshot 2). Che version: 5.3.0-SNAPSHOT OS and version: Ubuntu 16.04 LTS Docker version: 1.12.3 Che install: Docker container Additional information: Problem can be reliably reproduced, doesn't happen randomly: [Yes] № 1 in ver. 5.2.2 № 2 № 3 in ver. 5.2.2